docs/use/RestoreBackup.md

```diff
@@ -1,16 +1,28 @@
-
 # Restore the database from a previously saved backup
 
+Database backups and data replication are complementary components of an effective disaster recovery strategy. 
+
+Everest enables you to use backups to either create a new database or restore the backup data to the original database. Choosing between these two options often depends on the specific scenario, the urgency of recovery, the impact on ongoing operations, the need for data isolation, and the purpose of the restoration (such as testing, recovery from corruption, or analysis).
+
+## Restore database from backup
+
+Restoring to the same cluster is useful in the following scenarios:
+
+- **When restoring from a Point-in-time recovery (PITR) timestamp** as PITR does not currently support restoring to a new database.
+- **For faster recovery**: It's typically quicker to restore data to an existing database as it eliminates the need for creating a new database structure.
+- **Minimal disruption**: Restoring to the existing database ensures continuity in operations and minimizes disruptions since it doesn’t involve switching between different database instances.
```

Comment:
Actually I would argue the opposite, restoring to the same DB would disrupt the operation of that DB. :confused:

Reply:
I've removed it